engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -200,15 +200,21 @@ methodRef, args, def, getIdentifiedLocation(inputAst), meta(), diag()
       }
       case Tree.Assignment a -> {
         var reference = translateMethodReference(a.getPattern(), false);
+        var body = translateExpression(a.getExpr());
+        if (body == null) {
+            throw new NullPointerException();
```

Comment:
same comment as in the previous case

Reply:
As above. The stacktrace is the information.